gwvet/etg.py

```diff
@@ -84,7 +84,10 @@ def get_canonical_etg_name(name):
 # -----------------------------------------------------------------------------
 # register well known ETGs
 
-register_etg_parameters('omicron', time='peak')
+register_etg_parameters('omicron', **{
+    'time': 'peak',
+    'ylim': [10,5000],
```

Comment:
@tjma12, can you separate list elements with a space?

```suggestion
    'ylim': [10, 5000],
```